api/tests/app/delivery/test_send_to_providers.py

```diff
@@ -107,26 +107,26 @@ def test_should_send_personalised_template_to_correct_email_provider_and_persist
         personalisation={'name': 'Jo'}
     )
 
-    mocker.patch('app.aws_ses_client.send_email', return_value='reference')
+    mocker.patch('app.smtp_client.send_email', return_value=['reference', 'sent'])
 
     send_to_providers.send_email_to_provider(
         db_notification
     )
 
-    app.aws_ses_client.send_email.assert_called_once_with(
-        '"Sample service" <sample.service@test.notify.com>',
+    app.smtp_client.send_email.assert_called_once_with(
+        '"Sample service" <sample.service@notifytest.gov.au>',
```

Comment:
Is mail actually being sent out to this email?

Reply:
Nope, it's mocking it on line 110: https://github.com/govau/notify/pull/52/files#diff-c6288ac0b74b33c97c8b1882d7af6877R110 and returning fake values.